packages/agent/package.json

```diff
@@ -1,27 +1,27 @@
 {
-  "name": "@internal/agent",
-  "version": "0.0.1",
-  "private": true,
-  "main": "src/index.ts",
-  "dependencies": {
-    "@credo-ts/anoncreds": "0.5.1-alpha.51",
-    "@credo-ts/askar": "0.5.1-alpha.51",
-    "@credo-ts/cheqd": "0.5.1-alpha.51",
-    "@credo-ts/core": "0.5.1-alpha.51",
-    "@credo-ts/indy-vdr": "0.5.1-alpha.51",
-    "@credo-ts/openid4vc": "0.5.1-alpha.51",
-    "@credo-ts/question-answer": "0.5.1-alpha.51",
-    "@credo-ts/react-hooks": "0.6.1",
-    "@credo-ts/react-native": "0.5.1-alpha.51",
-    "@internal/utils": "*",
-    "@tanstack/react-query": "^4.33.0",
-    "credo-ts-didweb-anoncreds": "0.0.1-alpha.13",
-    "query-string": "^8.1.0",
-    "rxjs": "^7.8.1"
-  },
-  "peerDependencies": {
-    "@hyperledger/anoncreds-react-native": "*",
-    "@hyperledger/aries-askar-react-native": "*",
-    "@hyperledger/indy-vdr-react-native": "*"
-  }
+	"name": "@package/agent",
```

Comment:
Why @package over @internal?

Fine but never seen @package before

Reply:
`@package` is what we use for idcrypt.